doc/contribute/project_roles.rst

```diff
@@ -90,6 +90,22 @@ in addition to those listed for Contributors and Collaborators:
 * Responsibility to enforce the code of conduct.
 
 
+Role Retirement
+###############
+
+* Contributors elected to leadership roles (Maintainer,
```

Comment:
Remove this bullet and just explicit list the roles in the second bullet and remove the 'leadership role' term.

Reply:
Agreed, updating this